open-machine-learning-jupyter-book/_config.yml

```diff
@@ -24,6 +24,8 @@ execute:
     - 'ml-advanced/unsupervised-learning.ipynb'
     - 'data-science/data-science-in-the-cloud/the-azure-ml-sdk-way.ipynb'
     - 'llm/basic/transformer.ipynb'
+    - 'llm/basic/language-modelling.ipynb'
+    - 'llm/pretrained-model/pretraining-on-unlabeled-data.ipynb'

```

Comment:
the codes in the file need to train the model, and execution times out. i ran the outputs straight from the local.
![20240410162650](https://github.com/ocademy-ai/machine-learning/assets/112878645/797afdb0-0570-4a2a-8d23-e6bd72a268af)
![20240410162659](https://github.com/ocademy-ai/machine-learning/assets/112878645/981fa79d-95c0-43ed-abac-90a72dab7c64)

Reply:
You can just leave the llm chapters unloaded, refer to deep learning for details. in that case, remember to upload all the files with the output.